docs/introduction/open-source/public_code.md

```diff
@@ -92,108 +96,127 @@ Reviews MUST include source, policy, tests and documentation. | yes | repo is co
 
-- [x] documentation is good across the board, "recipies" would be nice
+- [ ] criterion met.
 
 Requirement | meets | links and notes
 -----|-----|-----
-All of the functionality of the codebase – policy as well as source – MUST be described in language clearly understandable for those that understand the purpose of the code. | yes | additional docs are generated from code comments
-The documentation of the codebase MUST contain: a description of how to install and run the source code, examples demonstrating the key functionality. | yes | [getting started](https://open-zaak.readthedocs.io/en/latest/development/getting_started.html), [post-install checklist](https://open-zaak.readthedocs.io/en/latest/installation/index.html#post-install-checklist), improvement: add demo fixtures/instructions
-The documentation of the codebase SHOULD contain: a high level description that is clearly understandable for a wide audience of stakeholders, like the general public and journalists, a section describing how to install and run a standalone version of the source code, including, if necessary, a test dataset, examples for all functionality. |  | [documentation](https://open-zaak.readthedocs.io/en/latest/introduction/index.html), would like to see standard recipies and examples of more fuctionality
-There SHOULD be continuous integration tests for the quality of the documentation. | yes | link checks, build checks, [Travis config](https://github.com/open-zaak/open-zaak/blob/main/.github/workflows/ci.yml#L178-L190)
-The documentation of the codebase MAY contain examples that make users want to immediately start using the codebase. |
-The code MAY be tested by using examples in the documentation. |
+All of the functionality of the codebase, policy as well as source, MUST be described in language clearly understandable for those that understand the purpose of the code. | yes | additional docs are generated from code comments
+The documentation of the codebase MUST contain a description of how to install and run the source code. | yes | [getting started](https://open-zaak.readthedocs.io/en/latest/development/getting_started.html), [post-install checklist](https://open-zaak.readthedocs.io/en/latest/installation/index.html#post-install-checklist)
+The documentation of the codebase MUST contain examples demonstrating the key functionality. |  | <!-- need to check --> improvement: add demo fixtures/instructions
+The documentation of the codebase SHOULD contain a high level description that is clearly understandable for a wide audience of stakeholders, like the general public and journalists. |  | [documentation](https://open-zaak.readthedocs.io/en/latest/introduction/index.html), would like to see standard recipes and examples of more functionality
+The documentation of the codebase SHOULD contain a section describing how to install and run a standalone version of the source code, including, if necessary, a test dataset. |  | <!-- need to check docs -->
```

Comment:
we have this in https://open-zaak.readthedocs.io/en/stable/installation/index.html#installation - which now points you to the target environment (real deploy, try out with docker-compose or install to actually contribute code to OZ)